fw/tls.c

```diff
@@ -107,6 +115,14 @@ tfw_tls_connection_recv(TfwConn *conn, struct sk_buff *skb)
 		return T_OK;
 	case T_OK:
 		/* A complete TLS record is received. */
+		switch (tls->conf->endpoint) {
+		case TTLS_IS_CLIENT:
+			TFW_INC_STAT_BH(clnt.tls_hs_successful);
```

Comment:
Looks like stat will be incremented for each skb, it implies for single handshake stat will be increased several times.

Reply:
Yep, my bad, fixed. Also client stats seem to be useless, deleted.